src/components/CoverPhoto/CoverPhoto.tsx

```diff
@@ -1,10 +1,11 @@
-import React from "react";
+import React, { ReactNodeArray } from "react";
 import Background from "../../pages/MyTrecipes/TrecipeCard/BetterDefaultImage.png";
 import PhotoUploader from "../PhotoUploader/PhotoUploader";
 
 export interface CoverphotoProps {
   trecipeCardID: number;
```

Comment:
I initially left trecipeCardId thinking we might need it to update image src for Trecipe Model, but on second thought, since we can probably do the same thing using callback method. Treating CoverPhoto component as display/photo upload component probably makes it reusable elsewhere too. I will remove this prop 👍